### DISCUSSION TITLE
02230179

### DISCUSSION THREAD

**Discussion by C-gyeltshen** [kevinlin1/just-the-class#37] [Q&A]
https://www.figma.com/file/OmiCf0PDWf7pwvXpjrj4TP/Untitled?type=design&node-id=171%3A2&mode=design&t=khz9bVSgz306XTnM-1

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
